layouts/partials/title.html

```diff
@@ -0,0 +1 @@
+{{ with .Params.FEP }}FEP {{ . }} – {{ end }}{{ .Title }}
```

Comment:
By the way, do we have any policies of having `\n` in the end of files or not?

Reply:
Having EOL at the end of file seems to be the universal standard. We should follow it unless we know exactly why we don't need EOL at some place.